.github/workflows/build.yml

```diff
@@ -91,17 +91,23 @@ jobs:
           pip install --upgrade --requirement requirements-test.txt
       - name: Set up pre-commit hook environments
         run: pre-commit install-hooks
-      - name: Run pre-commit on all files
+      - name: Create modified GitHub Actions pre-commit configuration
+        run: |
+          sed '/terraform_validate/d' < .pre-commit-config.yaml \
+          > /tmp/.github-actions-pre-commit-config.yaml
+      - name: Run pre-commit with modified configuration file on all files
         # We run pre-commit here with a custom configuration that has
         # the terraform-validate hook removed.  This is because
         # terraform validate cannot currently run without accessing
         # the remote state because of the way the providers are
-        # defined in terraform/providers.tf.  This is something that
-        # may be remedied in the future.  For more information, check
-        # out these two GitHub issues:
+        # defined in providers.tf.  This is something that may be
```

Comment:
This repo has `terraform/providers.tf`... you know what to do.

Reply:
It has been done. 77f6067.